build/rpmfc.c

```diff
@@ -248,23 +248,24 @@ static rpmds rpmdsSingleNS(rpmstrPool pool,
  * @param writeBytesLeft no. of bytes to feed to script on stdin
  * @param failNonZero	is script failure an error?
  * @param buildRoot	buildRoot directory (or NULL)
+ * @param dup		duplicate output to stream (or NULL)
  * @return		buffered stdout from script, NULL on error
  */     
-static StringBuf getOutputFrom(ARGV_t argv,
-                        const char * writePtr, size_t writeBytesLeft,
-                        int failNonZero, const char *buildRoot)
+static int getOutputFrom(ARGV_t argv,
```

Comment:
probably I would make this function to return rpmRC rather than (int), but up to you.

Reply:
Actually rpmRC is wrong for 90% of the uses it's used for, especially in the build code. It's only really meaningful for package/header read operations, for pretty much everything else int is better.